### PR TITLE
fix: use gitversion from launcher path in build

### PR DESCRIPTION
**Pull request by jithine**
## Context

gitversion is now available in launcher, no need to download it each time when used.

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by tkyi** [comment]
Maybe update README to indicate which scripts must be run in a SD build

**Review by adong — APPROVED** [review]

File: git-tag.sh [hunk 1]
```diff
@@ -4,15 +4,8 @@ DIR="$(cd "$(dirname "$0")" && pwd)"
...
-if [ ! -f "$GIT_VERSION" ] ; then
-    echo Downloading gitversion
-    wget -q -O - https://github.com/screwdriver-cd/gitversion/releases/latest \
-       | egrep -o '/screwdriver-cd/gitversion/releases/download/v[0-9.]*/gitversion_linux_amd64' \
-       | wget --base=http://github.com/ -i - -O /tmp/gitversion
-    chmod +x $GIT_VERSION
-fi
-
+# https://github.com/screwdriver-cd/gitversion/releases
+GIT_VERSION=/opt/sd/gitversion
```
Comment (adong) [inline git-tag.sh:8]: `/opt/sd/gitversion` I cannot find the reference in `https://github.com/screwdriver-cd/launcher/blob/master/Docker/run.sh`, where is the variable stored?

Comment (jithine) [reply git-tag.sh:8]: it's in PATH for SD build.

**Comment by jithine** [comment]
@tkyi all the scripts are supposed to run in SD build or in a linux environment. None works in your local. especially the gitversion

**Comment by tkyi** [comment]
Oh I thought you could run these locally: https://github.com/screwdriver-cd/toolbox#recreate-all-pipelines, https://github.com/screwdriver-cd/toolbox#re-tag-docker-images

**Comment by jithine** [comment]
@tkyi docker ones yes, but not the gitversion ones, since they are compiled only for linux, But i have restored previous behavior

**Review by tkyi — APPROVED** [review]

**Pull request merged**